### PR TITLE
feat: added getKeyByValue 

### PR DESCRIPTION
**Pull request by Noggling**
## Prerequisites

Please make sure you can check the following boxes:

-   [x] My code follows the code style of this project
-   [x] All new and existing tests passed
-   [ ] Fix any eslint/prettier warnings/errors: npm run lint
-   [x] Remove unnecessary console.log (use logInfo logWarn if it's needed)
-   [x] Are variable and function names properly describing what it does?

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

-   [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
-   [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] I have updated the documentation accordingly
-   [x] I have added tests to cover my changes

### Description

added getKeyByValue  and updated docs

### CODE REVIEW

**Review by jorgenholme96 — APPROVED** [review]
Har noen conflicts, men ser good ut 💯

File: src/utils/logger.ts [hunk 1]
```diff
@@ -94,12 +94,13 @@ export function logWarn(...args: any[]): void {
 export function logError(...args: any[]): void {
     localLog(LogType.Error, ...args);
 }
-
+/* istanbul ignore next */
```
Comment (khollund) [inline src/utils/logger.ts:97]: Hva betyr istanbul?

Comment (jorgenholme96) [reply src/utils/logger.ts:97]: Tror det ignorerer linjer for test coverage?

Comment (Noggling) [reply src/utils/logger.ts:97]: da ignorerer man tester i sammendraget ;) er noe jest greier ;)

Comment (khollund) [reply src/utils/logger.ts:97]: Ah 👍

**Pull request merged**